hr_attendance_report_theoretical_time/views/hr_employee_views.xml

```diff
@@ -7,6 +7,15 @@
             <field name="manual_attendance" position="after">
                 <field name="theoretical_hours_start_date"/>
             </field>
+            <button name="toggle_active" position="after">
+                <button name="%(hr_attendance_theoretical_employee_action)d"
+                        context="{'search_default_employee_id': active_id}"
+                        class="oe_stat_button"
+                        icon="fa-calendar"
```

Comment:
If you could choose another icon it would be great since it is the same as leaves' smart button.

Reply:
sure - which one ?

Reply:
'clock-o' for example?